test/integration/smoke/test_activate_template.py

```diff
@@ -107,14 +110,22 @@ def test_01_activate_sytem_vm_template(self):
         Test activating registered template
         """
         template = self.registerTemplate(self.test_template)
+        # Registering and Downloading System VM Template
         self.download(self.apiclient, template.id)
         self.activateTemplateCmd.id = template.id
+        # Activating downloaded template
         self.activateTemplate(self.activateTemplateCmd)
+        response = self.checkConfiguration(self.listConfigurationsCmd)
+        # Checking template activation
+        self.assertEqual(response[0].value, "test-system-kvm-4.11.3", "Expected template name to be test-system-kvm-4.11.3")
         return
 
-    def test_02_copySystemVMtemplate(self):
-        # this test needs 2 zones, the template will be copied from 1 zone to the next
-        pass
+    def test_02_get_default_url(self):
```

Comment:
This one is missing '@attr', without it it won't execute

Reply:
This test has been removed, so the comment is no longer relevant, but I added this attribute to the new test I added.

Reply:
This test has been removed, so the comment is no longer relevant, but I added this attribute to the new test I added.